### ISSUE TITLE
Kubernetes maximum 50 pod limit on latest 2.0.1.0 docker edge

### ISSUE THREAD

**Issue by bgehman** [docker/for-mac#3453]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [ ] I have uploaded Diagnostics
  - Diagnostics ID:

### Expected behavior
Able to deploy as many kubernetes PODs as I want.

### Actual behavior
Hitting what seems to be a 50-pod limit.  Exactly 50 PODs get to `Running` state, additional ones have this kubernetes event (from `kubectl describe pod -n <namespace> <podname>`:
```
Events:
  Type     Reason            Age                From               Message
  ----     ------            ----               ----               -------
  Warning  FailedScheduling  1m (x92 over 16m)  default-scheduler  0/1 nodes are available: 1 Insufficient pods.
```

### Information

  - macOS Version: 10.14.2
  - docker edge version: 2.0.1.0 (30090)

### Steps to reproduce the behavior
Try to deploy more than 50 PODs, only 50 get to running:

```
$ kubectl get pods --all-namespaces | grep Running | wc -l  
      50

```

**Comment by guillaumerose** [comment]
Interesting! Have you tried to add more CPU to your VM ?

**Comment by bgehman** [comment]
@guillaumerose It is not a CPU or RAM limitation -- seems to be a hard-coded POD limit.  Found this article: https://prefetch.net/blog/2018/02/10/the-kubernetes-110-pod-limit-per-node/ , but don't think we can override the `kubelet --max-pods` with how Docker is bundling Kubernetes (no option in the preferences).

I'm downgrading back to `2.0.0.0` where this was working fine prior to the upgrade.

**Comment by bgehman** [comment]
@guillaumerose Unlimited :) . But yeah, 200 should be plenty to stay out of the way.  50 is just too low -- especially as the internal kubernetes pods take up 9 of them, only leaving 41 for user-land.  Thanks.

**Comment by guillaumerose** [comment]
By default, it's 110. Let's go with 200.

https://github.com/kubernetes/kubernetes/blob/4fec22b3e4da6034948fd684f60637171cb55561/pkg/kubelet/apis/config/v1beta1/defaults.go#L158

**Comment by bgehman** [comment]
Just to confirm: I am seeing a 50-POD limit in v2.0.1.0 (which is just too low).
I do not see that lower limit in v2.0.0.0 -- I suspect it is 110 in that version...

Either the default 110, or increasing it to 200 would be sufficient from my POV.  Thanks.

**Comment by bgehman** [comment]
@guillaumerose Any explanation for why this was bug was closed?  Thanks.

**Comment by guillaumerose** [comment]
I merged the fix in our private repo. This was closed automatically, sorry.

The limit is removed, we let the default value. It will be inside the next release.

**Comment by bgehman** [comment]
Confirmed fixed in edge version `v2.0.2.0 (30972)`.
Thanks again, @guillaumerose 👍
